cosmwasm/README.md

```diff
@@ -1,45 +1,92 @@
-# Deploy
+# Intro
 
-First build the contracts
+This folder contains pyth contract on CosmWasm and utilities to deploy it in CosmWasm chains.
 
+# Deployment
+
+Deploying a contract in terra consists of two steps:
+1. Uploading the code. This step will give you a code id.
+2. Optionally create a new contract or migrate an existing one:
+    1. Creating a new contract which has an address with a code id as its program.
+    2. Migrating an existing contract code id to the new code id.
+
+This script can do both steps at the same time. Read below for the details.
+
+## Uploading the code
+
+First build the contracts:
 
 ``` sh
-docker build -f Dockerfile.build -o artifacts .
+bash build.sh
 ```
 
-Then, for example, to deploy `token_bridge.wasm`, run in the `tools` directory
+This command will builds and saves all the contracts in the `artifact` directory.
+
+Then, for example, to deploy `pyth_bridge.wasm`, run in the `tools` directory:
```

Comment:
why "for example"? Doesn't the user always want to deploy `pyth_bridge.wasm` ?

Reply:
If not, this should explain how to deploy all of the relevant contracts.

Reply:
this should also say "run the following command in ..."

Reply:
Correct.

The audience here is ourself :think: